### PR TITLE
Rename struct to imported name

### PR DESCRIPTION
**Pull request by fanatid**
Not required changes but makes code clear:
- `Response` imported as `RpcResponse` but still used in two places as `Response` (I thought that it's not possible!)
- `map` for `Result` not required

### CODE REVIEW

File: rpc/src/rpc.rs [hunk 1]
```diff
@@ -413,7 +413,7 @@ impl JsonRpcRequestProcessor {
                 })
                 .collect::<Result<Vec<_>>>()?
         };
-        Ok(result).map(|result| match with_context {
+        Ok(match with_context {
             true => OptionalContext::Context(new_response(&bank, result)),
             false => OptionalContext::NoContext(result),
```
Comment (CriesofCarrots) [inline rpc/src/rpc.rs:418]: Do you mind renaming this variable `result`, while you're in here?
That word is overloaded, so it's a little confusing. Maybe something like `accounts` instead?

**Review by CriesofCarrots — APPROVED** [review]
Thank you! One favor to ask of you, but these changes lgtm

File: rpc/src/rpc.rs [hunk 2]
```diff
@@ -113,7 +113,7 @@ const MAX_RPC_EPOCH_CREDITS_HISTORY: usize = 5;
 
 fn new_response<T>(bank: &Bank, value: T) -> RpcResponse<T> {
     let context = RpcResponseContext { slot: bank.slot() };
-    Response { context, value }
+    RpcResponse { context, value }
```
[the remark this answers is not in the document]
Comment (fanatid) [reply rpc/src/rpc.rs:116]: Indeed, that is why `Response` worked!

`result` renamed to `accounts`, thanks :+1:

**Review by CriesofCarrots — APPROVED** [review]

**Comment by CriesofCarrots** [comment]
Haha, oops, that one character was enough to catch the eye of rustfmt

**Comment by fanatid** [comment]
:smile: 

I check `clippy` before push but not `fmt` :man_facepalming:

**Comment by fanatid** [comment]
@CriesofCarrots any chance to re-review?

**Review by CriesofCarrots — APPROVED** [review]
lgtm, thanks for the nudge!

**Comment by fanatid** [comment]
Thank you! :heart:

**Pull request merged**